app/angular/package.json

```diff
@@ -81,7 +81,7 @@
     "@angular/forms": "^11.2.0",
     "@angular/platform-browser": "^11.2.0",
     "@angular/platform-browser-dynamic": "^11.2.0",
-    "@nrwl/workspace": "^11.1.5",
+    "@nrwl/workspace": "^11.6.3",
```

Comment:
This is an update to the devDependency.

This new version has some allowances that allow Nx to be tested _within_ an Nx repo.